server/.test.env

```diff
@@ -19,3 +19,4 @@ INPUTAPP_API_KEY=not-secret-key
 GLOBAL_WORKSPACE='mergin'
 GLOBAL_STORAGE=104857600
 COLLECT_STATISTICS=0
+GEODIFF_WORKING_DIR=/tmp/geodiff
```

Comment:
We need probably tune run_mergin to accept or not this variable, hm?

Reply:
True, this would be the case of any new introduced variable. On the other hand we have  sensible default so it is not a big deal.

Reply:
This could lead to problems, If someone is using mountpoint for projects data and is having limits for local machines tmp?

Reply:
I think we should rather document it properly else we end up with if-else branches in code, do this or do that which I wanted to avoid.

These kind of deployments problems CE is vulnerable even now, is not it?

Reply:
Yeah.. let's prepare PR to documentation :) It shoul be ok to maintain It there also :)